.github/workflows/release.yml

```diff
@@ -0,0 +1,69 @@
+# From: https://github.com/googleapis/google-cloud-go/blob/master/.github/workflows/release.yaml
+# This workflow creates release PRs and tags the top level release for
+# the top level repository:
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  release-please-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - id: release-pr # Open the release PR from a fork.
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          # token: ${{ secrets.FORKING_TOKEN }}
+          token: ${{ secrets.FORKING_TOKEN }}
```

Comment:
I'm curious what does this FORKING_TOKEN do? How does it differ from the GITHUB_TOKEN?

Reply:
I believe this is a separate secret used for forking the repo – I'm not sure why `google-cloud-go` does this. @bcoe do you have background?

Reply:
FORKING_TOKEN should be a token created from a GitHub account that's signed our CLA, and has no permissions on any sensitive repositories.

If you use `GITHUB_TOKEN` you will get CLA warnings.

Reply:
make sense, thank you!

@grant we can also use this for the auto-merge action script (https://github.com/googleapis/google-cloudevents/pull/212), this solves the CLA problem we had.

Reply:
@bcoe OK, I'll ask Yoshi Chat to create these tokens.

@jay-shi Logging in with this token may help with the CLA issue, where the action could run something like this before creating a PR.

```
gh auth login --with-token < mytoken.txt
```